presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -200,7 +210,7 @@ protected ResultSet getTables(Connection connection, Optional<String> schemaName
                 connection.getCatalog(),
                 escapeNamePattern(schemaName, escape).orElse(null),
                 escapeNamePattern(tableName, escape).orElse(null),
-                new String[] {"TABLE", "VIEW", "MATERIALIZED VIEW", "FOREIGN TABLE"});
+                tableTypes);
```

Comment:
`String[]` is not immutable:
```suggestion
                tableTypes.clone());
```